### PR TITLE
Add Nginx benchmark CI

### PR DESCRIPTION
**Pull request by sdww0**
This PR adds Nginx benchmark CI. Asterinas does not support the Nginx benchmark with concurrency > 1, so I only added the benchmark with the command `ab -n 10000 -c 1`.

### CODE REVIEW

**Comment by grief8** [comment]
@boterinas author

**Comment by sdww0** [comment]
@boterinas review

File: test/benchmark/nginx/http_req10k_conc1_bw/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "alert_threshold": "125%",
+    "alert_tool": "customBiggerIsBetter",
+    "search_pattern": "Transfer rate:",
+    "result_index": "3",
+    "description": "ab -n 10000 -c 1 http://localhost:8080/index.html",
+    "title": "[Network] Nginx HTTP request performance",
```
Comment (tatetian) [inline test/benchmark/nginx/http_req10k_conc1_bw/config.json:7]: Remove `[Network]`. Every benchmark results on Nginx is about _network_. So unlike LMbench, such a prefix prvoides no useful information at all.

Comment (sdww0) [reply test/benchmark/nginx/http_req10k_conc1_bw/config.json:7]: In addition to remove `[Network]`, I think it should also adding some information about the `-n 10000 -c 1`. So I change it to `Nginx HTTP request performance with 1 concurrency and 10000 requests in total`

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**